doc/source/visualization.rst

```diff
@@ -1245,6 +1245,16 @@ in ``pandas.plot_params`` can be used in a `with statement`:
 
    plt.close('all')
 
+Automatic Date Tick Adjustment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Especially for the ``TimedeltaIndex`` that uses native matplotlib
+tick locator methods, it is useful to call the automatic
+date tick adjustment from matplotlib for figures whose ticklabels overlap.
+
+See the :meth:`autofmt_xdate <matplotlib.figure.autofmt_xdate>` method and the
+`matplotlib documentation <http://matplotlib.org/api/figure_api.html#matplotlib.figure.Figure.autofmt_xdate>`__ for more.
```

Comment:
I would refer here to http://matplotlib.org/users/recipes.html#fixing-common-date-annoyances instead (the link to the api docs will also be generated through the `:meth:..` you did, and is also included in this recipe section.